cmp_compressonatorlib/bc6h/codec_bc6h.cpp

```diff
@@ -160,76 +147,59 @@ bool CCodec_BC6H::SetParameter(const CMP_CHAR* pszParamName, CMP_DWORD dwValue)
 
 bool CCodec_BC6H::SetParameter(const CMP_CHAR* pszParamName, CODECFLOAT fValue)
 {
-    if (strcmp(pszParamName, "Quality") == 0)
-        m_Quality = fValue;
-    //else
-    //if (strcmp(pszParamName, "OutputExposure") == 0)
-    //     m_Exposure = fValue;
-    else
-        return CCodec_DXTC::SetParameter(pszParamName, fValue);
-    return true;
+    return CCodec_DXTC::SetParameter(pszParamName, fValue);
 }
 
 CCodec_BC6H::~CCodec_BC6H()
 {
     if (m_LibraryInitialized)
     {
-        if (m_Use_MultiThreading)
```

Comment:
I'm not happy with the diff here, its hard to read and makes me think I missed something. I'm going to tidy this up in a more SOLID/DRY way so that thread creation and destruction is a bit more modularized here and in the other locations it is edited in.

I'll also see if I can find tests for it.